resources/lang/ca/group.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+return [
+    'group'                         => 'Group',
+    'id'                            => 'Id',
+    'status'                        => 'Status',
+    'vehicle'                       => 'Vehicle',    
+    'broadcast'                     => 'Broadcast',
+    'gps'                           => 'GPS',
+    'people_involved'               => 'People Involved',
+    'status_active'                 => 'Active',
+    'status_closed'                 => 'Closed',
+    'status_active_value'           => '0',
+    'status_closed_value'           => '1',
+    'lengthMenu'                    => 'Show _MENU_ groups',
+    'group_add_success'             => 'Group was successfully added !',
+    'group_edit_success'            => 'Group was successfully edited !',
+    'group_delete_success'          => 'Group was successfully deleted !',
+    'group_not_found'               => 'Group not found.',
+    'unable_to_delete'               => 'Unable to delete.',
```

Comment:
This is not catalan. You should only add the English version or the ones that you know for sure are correct.